shared/src/main/scala/drt/shared/HasAirportConfig.scala

```diff
@@ -247,8 +249,32 @@ case class AirportConfig(portCode: PortCode,
                          hasTransfer: Boolean = false,
                          maybeCiriumEstThresholdHours: Option[Int] = None,
                          maybeCiriumTaxiThresholdMinutes: Option[Int] = Option(20),
-                         feedSources: Seq[FeedSource] = Seq(LiveBaseFeedSource, LiveFeedSource, AclFeedSource, ApiFeedSource)
+                         feedSources: Seq[FeedSource] = Seq(LiveBaseFeedSource, LiveFeedSource, AclFeedSource, ApiFeedSource),
+                         desksByTerminal: Map[Terminal, Int],
+                         flexedQueuesPriority: List[Queue] = List(EeaDesk, NonEeaDesk, QueueDesk, FastTrack)
                         ) {
+  def assertValid(): Unit = {
```

Comment:
do we have a test that calls this on all the airport configs in the app?

Reply:
It gets called on startup of the app, so the app will fail if it fails.
Could add a test too